tests/integration.rs

```diff
@@ -100,7 +100,11 @@ fn single_empty_file(ext: Extension, #[any(size_range(0..8).lift())] exts: Vec<F
 
 // compress and decompress a single file
 #[proptest(cases = 512)]
-fn single_file(ext: Extension, #[any(size_range(0..8).lift())] exts: Vec<FileExtension>) {
+fn single_file(
+    ext: Extension,
+    #[any(size_range(0..8).lift())] exts: Vec<FileExtension>,
+    #[strategy(0i16..30)] level: i16,
```

Comment:
```suggestion
    #[strategy(0i16..30)] level: Option<i16>,
```
and put the `ouch -A c` line in an `if let` statement

Reply:
Didn' t find how to the strategy with Option.

Reply:
this will work
```suggestion
    #[strategy(proptest::option::of(0i16..30))] level: Option<i16>,
```

and you would need to change the below to this
```rust
    if let Some(level) = level {
        ouch!("-A", "c", "-l", level.to_string(), before_file, archive);
    } else {
        ouch!("-A", "c", before_file, archive);
    }
```